### PR TITLE
feat: add config for readthedocs

### PR DESCRIPTION
**Pull request by jpetot**

### CODE REVIEW

**Comment by jpetot** [comment]
https://saiph.readthedocs.io/en/latest/ 🎉 📖

**Comment by tcrasset** [comment]
image
show**s**

**Review by tcrasset — APPROVED** [review]
Nice! Good job!

File: .readthedocs.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.9"
+    # You can also specify other tool versions:
+    # nodejs: "16"
+    # rust: "1.55"
+    # golang: "1.17"
```
Comment (charlax) [inline .readthedocs.yaml:16]: Perhaps worth removing?

File: .readthedocs.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+
+# If using Sphinx, optionally build your docs in additional formats such as PDF
+# formats:
+#    - pdf
```
Comment (charlax) [inline .readthedocs.yaml:24]: Same :)

File: .readthedocs.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+# Optionally declare the Python requirements required to build your docs
```
Comment (charlax) [inline .readthedocs.yaml:26]: Remove too?

**Pull request merged**